company-yasnippet.el

```diff
@@ -97,6 +97,22 @@
        res))
    tables))
 
+(defun company-yasnippet--doc (arg)
+  (let ((template (get-text-property 0 'yas-template arg))
+        (mode (format "%s" major-mode)))
```

Comment:
Why not keep `mode` a symbol value? Then, below you will compare it with `'web-mode` and set to `'html-mode` if true.

Reply:
thanks.I'm not very good at writing elisp.

Reply:
How about:

```
(mode major-mode)
```

Reply:
got it! thanks.